tests/test_default_settings.py

```diff
@@ -0,0 +1,109 @@
+import logging
+from pathlib import Path
+
+import jsonschema
+import requests
+import yaml
+from copier.main import copy
+from plumbum import local
+
+logger = logging.getLogger(__name__)
+
+
+def get_schema():
+    """This function loads the given schema available"""
+    schema_url = "https://json.schemastore.org/github-workflow.json"
+    schema_path = "github-workflow.json"
+    # Try to fetch directly from server to get latest version
+    try:
+        logger.info(f"Trying to fetch data from {schema_url}")
+        data = requests.get(schema_url).content
+    except requests.exceptions.RequestException:
+        logger.info(f"Fetching from URL failed, using file {schema_path}")
+        with open(schema_path, "r") as file:
+            data = file.read()
+    schema = yaml.safe_load(data)
+    return schema
+
+
+def validate_schema(yaml_data):
+    """Validate yaml based on schema."""
+    expected_schema = get_schema()
+    try:
+        jsonschema.validate(instance=yaml_data, schema=expected_schema)
+    except jsonschema.exceptions.ValidationError as err:
+        logger.error(err)
+        return False
+    return True
```

Comment:
I don't see much sense on slurping the exception to print the error manually... just let the exception raise and pytest will print the full error stack.

```suggestion
    jsonschema.validate(instance=yaml_data, schema=expected_schema)
```